### ISSUE TITLE
Question about learning rate while resuming training

### ISSUE THREAD

**Issue by starspder** [AlexeyAB/darknet#8594]
Hi I have a question about learning rate

Assume YOLOv4 cfg...
ex)) steps: 25000
       scales: .1
       max_batches = 400,000
       learning_rate=0.001/2=0.0005 # learning_rate / gpu_num 

If I stop training 300,000 times and put in last_weights and resume, at that time, learning_rate=0.0005*0.1=0.00005

In other words, I wonder if learning_rate=0.00005 should be changed In cfg again or if learning_rate is saved at that time In last_weights file so learning is automatically performed at 0.0005.

Thank you for reading my question!
